apps/meteor/app/livechat/server/lib/analytics/dashboards.ts

```diff
@@ -247,11 +247,11 @@ const getConversationsMetricsAsync = async ({
 			language: user.language || settings.get('Language') || 'en',
 		})) || [];
 	const metrics = ['Total_conversations', 'Open_conversations', 'On_Hold_conversations', 'Total_messages'];
-	const visitorsCount = await LivechatVisitors.getVisitorsBetweenDate({
```

Comment:
it seems `getVisitorsBetweenDate` is not used anywhere else, what about removing it? or actually just rename it to the new one

Reply:
Gonna remove